src/GitVersionCore/GitVersionCore.csproj

```diff
@@ -47,7 +47,9 @@
       <HintPath>..\packages\LibGit2Sharp.0.23.0-pre20160922233542\lib\net40\LibGit2Sharp.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="System" />
+    <Reference Include="System">
+      <HintPath>..\..\..\..\..\..\..\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.0\System.dll</HintPath>
```

Comment:
I think you don't need to reference assembly by explicit path.

Reply:
Bah!  I blame visual studio for this abomination!  Thanks.  I will just revert this to the original line.